src/components/ha-sidebar.ts

```diff
@@ -980,7 +980,7 @@ class HaSidebar extends LitElement {
           line-height: 20px;
           text-align: center;
           padding: 0px 6px;
-          color: var(--text-accent-color, var(--text-primary-color));
+          color: var(--text-accent-color, #202020);
```

Comment:
This should stay unchanged, we should set a default fro the `text-accent-color` variable in the light and dark theme.
```suggestion
          color: var(--text-accent-color, var(--text-primary-color));
```